docs/primitives/cron-jobs.md

```diff
@@ -78,6 +78,8 @@ the `Every` field is not expressive enough.
 For these use cases, Encore provides full support for [Cron expressions](https://en.wikipedia.org/wiki/Cron) by using the `Schedule` field
 instead of the `Every` field.
 
+Cron expressions allow you to define precise schedules for your tasks, including specific days of the week, specific months, and more. When using Cron expressions in Encore, it's important to note that they operate based on Coordinated Universal Time (UTC). Therefore, when defining Cron expressions, ensure that you account for the time zone offset if necessary.
```

Comment:
Thanks! I think we can make this more succinct:
```suggestion
Cron expressions allow you to define precise schedules for your tasks, including specific days of the week, specific hours of the day, and more. Note that all times are expressed in UTC.
```

Reply:
Seems good! Its now clear and succinct.